examples/secondary-care/homecare/acute/nominated-pharmacy/clinical-practitioner/4-Convert-Response-Claim-200_OK.xml

```diff
@@ -41,6 +41,9 @@
     <subject>
       <DispenseClaimInformation classCode="INFO" moodCode="EVN">
         <id root="A2C05EC6-4292-4568-8200-BFAC8439DA04"/>
+        <code code="163541000000107" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Dispensed Medication - FocusActOrEvent (administrative concept)"/>
+        <effectiveTime value="PLACEHOLDER"/>
```

Comment:
should this placeholder be being replaced?

Reply:
yep, will sort